rero_ils/modules/items/api/circulation.py

```diff
@@ -1251,18 +1250,20 @@ def available(self):
         """Get availability for item."""
         return self.item_has_active_loan_or_request() == 0
 
-    def get_item_end_date(self, format='short', time_format='medium'):
+    def get_item_end_date(self, format='short', time_format='medium',
+                          language=None):
```

Comment:
would be nice if you can add :param and :return in the method description

Reply:
Done